schema/deploy/mutations/add_contact_to_revision.sql

```diff
@@ -24,7 +25,8 @@ as $add_contact_form_change$
     'project_contact',
     $1,
     'pending',
-    'project_contact'
+    'project_contact',
+    '[{"message": "must have required property contactId"}]'
```

Comment:
This need a unit test (contacts added to a revision should have validation errors)

Reply:
Added!